README.md

```diff
@@ -86,30 +88,7 @@ $ gulp test
 $ npm test
```

Comment:
Do you think Prerequisites step 3 should be removed since it's describing creating a new API? We might want to move this to a different section in the skeleton readme also since it's a little different than the other steps in this section